build/build-report.js

```diff
@@ -96,8 +96,32 @@ async function buildEsModulesBundle() {
       rollupPlugins.commonjs(),
       // Exclude this 30kb from the devtools bundle for now.
       rollupPlugins.shim({
-        [`${LH_ROOT}/shared/localization/i18n-module.js`]:
-            'export const swapLocale = _ => {}; export const format = _ => {};',
+        [`${LH_ROOT}/shared/localization/i18n-module.js`]: `
```

Comment:
Seems ridiculous but I can't think of a better way to do accomplish this. Better to be ridiculous than to overengineer it :)

For readability, would this string be better in a variable than inline? Alternatively should we make a `build/shims/` directory and `rollupPlugins.alias()` `i18n-module.js` to point there? I'm assuming that would work but it may not

Reply:
a `shims/i18nmodule.js` sg. then it's at least lintable

Reply:
i tried a shim file and it's terrible. ends up creating more cruft.  so.. leaving as is.